src/mudlet.cpp

```diff
@@ -3671,6 +3646,36 @@ void mudlet::startAutoLogin()
     }
 }
 
+bool mudlet::addHost(const QString& hostname, const QString& port, const QString& login, const QString& pass)
+{
+    auto success = mHostManager.addHost(hostname, port, login, pass);
```

Comment:
We now have `HostManager::addHost()` and `mudlet::addHost()` and the intention is that it's only the latter that should be called, because it wraps some necessary initialisation (the central debug area) around the method once we have our first host.

I'm not sure how to properly describe it using C++ semantics.

Reply:
Make only the latter callable?

Reply:
The `mudlet` class is *[overriding](https://en.wikipedia.org/wiki/Method_overriding#C++)* the `addHost` method of the `HostManager`  class maybe? 🤓

Reply:
In retrospect, this design was bad. Fixed it to a more logical one - you call `HostManager.addHost` as you did before now.